e2e/dractions/disapp.go

```diff
@@ -96,16 +96,25 @@ func DisableProtectionDiscoveredApps(ctx types.TestContext) error {
 		return err
 	}
 
+	// delete placement
+	if err := deployers.DeletePlacement(ctx, placementName, managementNamespace); err != nil {
+		return err
+	}
+
+	err = deployers.DeleteManagedClusterSetBinding(ctx, config.ClusterSet, managementNamespace)
+	if err != nil {
+		return err
+	}
+
 	if err := waitDRPCDeleted(ctx, managementNamespace, drpcName); err != nil {
 		return err
 	}
 
-	// delete placement
-	if err := deployers.DeletePlacement(ctx, placementName, managementNamespace); err != nil {
+	if err := deployers.WaitPlacementDeleted(ctx, placementName, managementNamespace); err != nil {
 		return err
 	}
 
-	err = deployers.DeleteManagedClusterSetBinding(ctx, config.ClusterSet, managementNamespace)
+	err = deployers.WaitManagedClusterSetBindingDeleted(ctx, config.ClusterSet, managementNamespace)
```

Comment:
This style inconsistency was already here, we need to unify the style. Maybe add another commit to unify it?